Jigsaw-Code/outline-server
# Restarts Prometheus subprocess when it crashes

Adds restart logic for the Prometheus subprocess by listening for process exit events.

Manually tested using the following:
$ yarn do shadowbox/docker/run
$ pgrep prometheus | xargs sudo kill

TODO: Can we write an integration test?
TODO: Can we publish an event somewhere to track the number of crashes we see?
